packages/app/vite-utils/vite.base.config.ts

```diff
@@ -48,6 +49,28 @@ const baseConfig: UserConfig = {
   define: {
     'process.env': {},
   },
+  ...(process.env.WITH_PNPM_LINKS && {
+    resolve: {
+      alias: {
+        '@fuel-ui/react': path.resolve(
+          __dirname,
+          '../node_modules/@fuel-ui/react/dist/index.mjs'
+        ),
+        '@fuel-ui/css': path.resolve(
+          __dirname,
+          '../node_modules/@fuel-ui/css/dist/index.mjs'
+        ),
+        '@fuel-ui/config': path.resolve(
+          __dirname,
+          '../node_modules/@fuel-ui/config/dist/index.mjs'
+        ),
+        '@fuel-ui/test-utils': path.resolve(
+          __dirname,
+          '../node_modules/@fuel-ui/test-utils/dist/index.mjs'
+        ),
+      },
+    },
+  }),
```

Comment:
We could probably DRY this a bit by using regexes.
 - https://regexr.com/7790c

It should look something like the following.

```suggestion
  ...(process.env.WITH_PNPM_LINKS && {
    resolve: {
      alias: [{
        find: /(@?fuels?-?[^\s]*)/,
        replacement: path.resolve(__dirname, '../node_modules/$1/dist/index.mjs'
      }],
    },
  }),
```